resources/js/chat_menu.js

```diff
@@ -124,7 +124,7 @@ export default class ChatMenu extends React.Component {
 					className="avatar"
 					key="Everyone"
 					alt="Everyone"
-					src="/static/un_logo.svg"
+					src="/static/img/un_logo.svg"
```

Comment:
Great :)

At some point we'll want to move the js files to static/js, I guess.